packages/xo-server-test/src/_xoConnection.js

```diff
@@ -128,7 +147,7 @@ class XoConnection extends Xo {
 
   async createTempVm(params) {
     const id = await this.call('vm.create', {
-      name_label: 'XO Test',
+      name_label: getDefaultName(),
```

Comment:
Does it not break existing tests?

Reply:
No, there not have a test which store a snapshot of the method output.